packages/eslint-config-hmpps/index.js

```diff
@@ -0,0 +1,147 @@
+const _import = require("eslint-plugin-import")
+const noOnlyTests = require("eslint-plugin-no-only-tests")
+
+const { fixupPluginRules } = require("@eslint/compat")
+
+const globals = require("globals")
+const typescriptEslint = require("@typescript-eslint/eslint-plugin")
+const tsParser = require("@typescript-eslint/parser")
+const js = require("@eslint/js")
+
+const { FlatCompat } = require("@eslint/eslintrc")
+
+const { rules: bestPracticesRules } = require('./rules/best-practices')
+const { rules: errorsRules } = require('./rules/errors')
+const { rules: nodeRules } = require('./rules/node')
+const { rules: styleRules } = require('./rules/style')
+const { rules: variablesRules } = require('./rules/variables')
+const { rules: es6Rules } = require('./rules/es6')
+const { rules: importsRules } = require('./rules/imports')
+const { rules: strictRules } = require('./rules/strict')
+
+const airBnbRules = Object.assign({}, bestPracticesRules, errorsRules, nodeRules, styleRules, variablesRules, es6Rules, importsRules, strictRules)
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+})
+
+module.exports = [
+  {
+    ignores: [
+      "**/node_modules",
+      "**/public",
+      "**/assets",
+      "**/cypress.json",
+      "**/reporter-config.json",
```

Comment:
why ignore these 2 files? since other json files are checked, it makes sense to enforce style here too